src/components/modal/v2/parts/views/NI/Content.jsx

```diff
@@ -0,0 +1,71 @@
+/** @jsx h */
+import { h, Fragment } from 'preact';
+import { useState, useRef } from 'preact/hooks';
+import Button from '../../Button';
+import Icon from '../../Icon';
+import ProductListLink from '../../ProductListLink';
+import Instructions from '../../Instructions';
+import styles from './styles/index.scss';
+import { useApplyNow } from '../../../lib';
```

Comment:
```suggestion
import { useServerData, useApplyNow } from '../../../lib';
```
We can import `useServerData` here so that we can use it in a check for rendering the product list list or not.